pandas/io/common.py

```diff
@@ -519,6 +519,21 @@ def infer_compression(
     raise ValueError(msg)
 
 
+def check_parent_directory(fpath: str):
```

Comment:
call this `path`, why isn't this Path | str?

returns -> None

Reply:
done